### PR TITLE
add an api usage example

### PR DESCRIPTION
**Pull request by devoncarew**
- add an api usage example
- fix an `http` reference in the changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -201,7 +202,7 @@
 # 1.0.0
 
 This package was extracted from the
-[`source_maps`](http://pub.dartlang.org/packages/source_maps) package, but the
+[`source_maps`](https://pub.dartlang.org/packages/source_maps) package, but the
```
Comment (kevmoo) [inline CHANGELOG.md:205]: pub.dev?

Comment (kevmoo) [reply CHANGELOG.md:205]:
```suggestion
[`source_maps`](https://pub.dev/packages/source_maps) package, but the
```

File: pubspec.yaml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 name: source_span
 version: 1.9.2-dev
-description: A library for identifying source spans and locations.
+description: >
```
Comment (kevmoo) [inline pubspec.yaml:3]:
```suggestion
description: >-
```

https://yaml-multiline.info/

File: example/main.dart [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+  final sourceFile = SourceFile.fromString(contents, url: file.uri);
+  final spans = parseFile(contents, sourceFile);
+
+  for (var span in spans.take(30)) {
+    print('[${span.start.line + 1}:${span.start.column + 1}] ${span.text}');
+  }
+}
+
+Iterable<SourceSpan> parseFile(String contents, SourceFile sourceFile) sync* {
```
Comment (kevmoo) [inline example/main.dart:21]: make this private, too?

**Review by kevmoo — APPROVED** [review]

File: example/main.dart (hunk 3 above)
Comment (devoncarew) [reply example/main.dart]: sgtm

**Pull request merged**